test-suite/coq-makefile/coqdoc1/run.sh

```diff
@@ -9,7 +9,13 @@ make html mlihtml
 make install DSTROOT="$PWD/tmp"
 make install-doc DSTROOT="$PWD/tmp"
 #make debug
-(for d in `find tmp -name user-contrib` ; do pushd $d >/dev/null && find . && popd >/dev/null; done) | sort -u > actual
+(
+  while IFS= read -r -d '' d
+  do
+    pushd "$d" >/dev/null && find . && popd >/dev/null
+  done < <(find tmp -name user-contrib -print0)
```

Comment:
I think (my personal opinion) non posix constructs like <() deserve a small comment like 

for <(cmd) see https://www.gnu.org/software/bash/manual/html_node/Process-Substitution.html)

It is tricky to google something like <() and it is not that widely used. Code should be optimized for readability by average experienced engineers.

Reply:
P.s.: Inst't there a more concise way to write this and still handle paths with spaces in it properly?

Reply:
Added a comment. I have no better idea at the moment...